src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1672,5 +1672,70 @@ public SeparatedSyntaxList<SyntaxNode> GetArgumentsForInvocationExpression(Synta
         {
             return ((invocationExpression as InvocationExpressionSyntax)?.ArgumentList.Arguments).Value;
         }
+
+        public bool IsDocumentationComment(SyntaxNode node)
+        {
+            return SyntaxFacts.IsDocumentationCommentTrivia(node.Kind());
+        }
+
+        public bool IsDirectiveOrImport(SyntaxNode node)
```

Comment:
Name is pretty vague.  Lot of things are called 'directives' (for example, preprocessor directives).  You could be more explicit and say IsUsingExternOrImportDirective